themes/qgrid/source/css/icons.scss

```diff
@@ -0,0 +1,26 @@
+@import "variables";
+
+.icon {
+  display: inline-block;
```

Comment:
is is settings for header icons if so maybe good idea to put appropriate style to head itself?

Reply:
I'll remove grid-area from icons - styles to meke them more common